scripts/test-local.sh

```diff
@@ -12,7 +12,7 @@ if [ -z "$GLOB" ]; then
   GLOB="test/local test/remote"
 fi
 
-DEFAULT_ARGS="-R dot --recursive --timeout 5000"
+DEFAULT_ARGS="-R dot --recursive --timeout 5000 --exit"
```

Comment:
I got this to work without `--exit` by adding `redis.close()` in certain places + some other stuffs. 

Using this:
https://gist.github.com/boneskull/7fe75b63d613fa940db7ec990a5f5843